examples/01_getting_started.py

```diff
@@ -0,0 +1,87 @@
+import os
+
+from haystack.document_stores import InMemoryDocumentStore
+from haystack.nodes import AnswerParser, PromptNode, PromptTemplate, BM25Retriever
+from haystack.pipelines import Pipeline
+from haystack.pipelines.standard_pipelines import TextIndexingPipeline
+from haystack.utils import fetch_archive_from_http
+from haystack.utils import print_answers
+
+
+def getting_started():
+    # This getting_started shows you how to use LLMs with your data, a technique called Retrieval Augmented Generation - RAG
+
+    # We are model agnostic :) Here, you can choose from: "Anthropic", "Cohere", "Huggingface", "OpenAI".
+    provider = "OpenAI"
+    # Please add an API key matching the provider.
+    API_KEY = "ADD KEY HERE"
+
+    # Loads good default pipelines for each provider.
+    indexing_pipeline, query_pipeline = load_pipelines(provider=provider, API_KEY=API_KEY)
+
+    # Downloads and adds a few Game of Thrones wiki articles to haystacks database.
+    # You can also provide a folder with your local documents. Be aware that some of your data will be sent to external APIs!
+    add_example_data(indexing_pipeline, dir="data/getting_started")
```

Comment:
index_example_data() ?

Reply:
Id like to use a word that everybody understands. "indexing" is a little bit domain lingo (although I agree that most haystack users can understand what indexing means). Additionally: this function downloads + indexes, thats why I thought it is nice to say "add".